lib/config.js

```diff
@@ -0,0 +1,7 @@
+const awsConfig = {
+  accessKeyId: process.env.AWS_ACCESS_KEY_ID,
+  secretAccessKey: process.env.AWS_SECRET_ACCESS_KEY,
+  region: "us-west-2",
+};
```

Comment:
when this pr gets merged can u send a msg into slack dev for the updated env variables